### DISCUSSION TITLE
Topological spaces among US

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#450] [General]
This is easily [my most popular Math.SE question](https://math.stackexchange.com/questions/4778063/is-there-anyone-among-us-who-can-identify-a-certain-sus-space), though not due to the mathematical content I'm afraid...

[Butterfly meme - Reaching for "inserting among us puns into my research" - Caption: "is this math outreach"](https://github.com/pi-base/data/assets/1559632/022cd759-09bc-40ef-8013-2ba82e3493ee)

The "SUS" property came out of Alan Dow's comments the other day after my talk, though the memes are completely original. I raised it as a thing during today's Pitt seminar presentation I gave, and Paul Gartside suggested a more natural property between $k_2H$ and $US$ might consider one-point compactifications more generally, rather than the limit of a "long sequence". I need to work out details, but I think this would live between $k_2H$ and SUS.  EDIT: (SUS will herefore be defined as USR, for "Unique Strongly Radial limits")

I decided to start a thread because I've been really enjoying doing my math in the open on Math.SE and MO, but this is a space where I can do so without quite as much concern for moderation. So I'm going to try working out some details on this property (and a few others Gartside suggested) in this thread; others are welcome to participate as they like (or mute the thread).

**Comment by StevenClontz** [comment]
### Definition

A space $X$ is **UK** ("Unique Kompactifying limits") if for every non-compact locally compact Hausdorff $L$ and continuous function $f:L\to X$, there exists at most one **compactifying limit** $x\in X$ such that for each neighborhood $N$ of $x$, there exists co-compact $C\subseteq L$ with $f[C]\subseteq N$.

### Proposition

If $f:L\to X$ is continuous and $x\in X$ is a compactifying limit, then $f[L]\cup\{x\}$ is compact.

**Proof:** Let $K=L\cup\{k\}$  Let $L$ be open with its usual neighborhoods, and let neighborhoods of $k$ be co-compact subsets of $L$. It follows that $K$ is compact Hausdroff.

Now let $f':K\to X$ be defined by $f'(l)=f(l)$ for $l\in L$, and $f'(k)=x$. We claim $f'$ is continuous and thus $f[K]=f[L]\cup\{x\}$ is compact. To see this, let $U$ be open in $X$. If $x\not\in U$, then by continuity of $f$ we have the inverse image of $U$ is open. If $x\in U$, choose co-compact $C\subseteq L$ with $f[C]\subseteq U$. Then $\{k\}\cup C\subseteq K$ is co-compact and thus a neighborhood of $k$ with $f'[\{k\}\cup C]\subseteq U$, and we're done.

### Theorem

All $k_2H$ spaces are UK.

**Proof:** Consider $f:L\to X$ continuous such that there exist points $x,y\in X$ such that for each pair of neighborhoods $N_x,N_y$ of $x,y$, there exist co-compact sets $C_x,C_y\subseteq L$ with $f[C_x]\subseteq N_x,f[C_y]\subseteq N_y$. We define $g:K\times\\{0,1\\}\to X$ by $g(l,i)=f(l)$, $g(\infty,0)=x$, and $g(\infty,1)=y$ and note $g$ is continuous. Pick open neighborhoods of $(\infty,0),(\infty,1)$, and note their images must intersect. Thus by $k_2H$ we have $x=y$.

**Reply by StevenClontz** [reply]
### Definition

A **transfinite sequence** is a map from an infinite cardinal to the space.

### Definition

A space is **USR** ("Unique Strongly Radial limits") provided the limit of every countinous transfinite sequence is unique.

### Proposition 

Every UK space is USR, and every USR space is US.

**Proof:** Every limit ordinal is a non compact, locally compact Hausdorff space, and a standard sequence is a continuous transfinite sequence.

**Reply by StevenClontz** [reply]
EDIT: none of this was good (not deleting in case the edit history is helpful though)

**Comment by prabau** [comment]
Interesting stuff.  The definition of gwH may not be completely clear, at least to me.  The idea seems to be that you want to consider one-point compactifications $L\cup\\{\infty\\}$ and consider $x$ as a limit as points of $L$ tend to $\infty$.  So $L$ should also be specified as non-compact, right?  Also if we want to implicitly talk about $x=\lim_{z\to\infty}f(z)$ with $z\in L$, shouldn't it be that each nbhd ot $x$ should contain $f[L\setminus K]$ for some compact $K\subseteq L$?  That does not seem to be the same as containing a co-compact subset of $f[L]$?

**Reply by StevenClontz** [reply]
Yeah, those were the issues that came to me last night when I decided to put a pin in things and get ready for bed. :-) I'll chew on this and make an edit above.

**Comment by StevenClontz** [comment]
Looking into spaces that aren't radial to find a USR-not-k2H example. Unfortunately this one turns out to be k1H, but it's still interesting I think.

Let $X=(\omega_1\times\omega)\cup\\{\infty_0,\infty_1\\}$ have $\omega_1\times\omega$ with the product of their order topologies, and neighborhoods of $\infty_i$ are of the form $\\{\infty_i\\}\cup(\omega_1\setminus\alpha)\times(\omega\setminus n)$.

$X$ is $k_1H$ because compacts are subsets of some Hausdroff subspace $(\alpha\times n)\cup\\{\infty_0,\infty_1\\}$. But I was hoping it would end up being more interesting, because long sequences cannot converge to either $\infty_i$.

**Comment by StevenClontz** [comment]
[M W](https://math.stackexchange.com/q/4828939) came to the rescue and pointed out a couple things.

### Definition

A space is UR ("Unique Radial limits") provided any limit of a transfinite sequence is unique.

### Proposition

$T_2\Rightarrow UR\Rightarrow USR$

**Proof:** standard arguments.

### Example

The Fortissimo space on $\omega_1$ with a doubled extra point is USR but not UR. This is the space $\omega_1\cup\{\infty_1,\infty_2\}$ where neighborhoods of either $\infty$ are co-countable and $\omega_1$ is discrete. Then the identity of $\omega_1$ is a transfinite sequence with two limits, but the space is anticompact and $T_1$, and thus $k_1H$ and $USR$.

**Reply by StevenClontz** [reply]
## Proposition

Let $X$ be a locally compact, compact Hausdorff space. Let $Y=X\cup\{x'\}$ where $x'$ is a double of some non-isolated $x\in X$. Then $Y$ is not GwH.

## Proposition

Let $X$ be a sequentially discrete space. Let $Y=X\cup\{x'\}$ where $x'$ is a double of some non-isolated $x\in X$. Then $Y$ is SUS. 

**Proof:** note that all convergent strict transfinite sequences are trivial, and thus have unique limits.

## Example

Let $X=\beta\omega$ and pick $x\in\beta\omega\setminus\omega$. Then $Y=X\cup\{x'\}$ is SUS but not GwH.

**Reply by StevenClontz** [reply]
### Proposition

Every locally countable US space is UR.

**Proof:**

Follows from the arguments of https://math.stackexchange.com/questions/4785088/ 

### Example 

The one-point compactification of the rationals is UR but not $T_2$ (or even $k_1H$).

**Reply by prabau** [reply]
Thinking out loud about the differences between the definitions.  Given an infinite cardinal $\kappa$, we know what it means to say a transfinite sequence (also called $\kappa$-sequence) $(x_\alpha)_{\alpha<\kappa}$ in $X$ converges to some limit $a\in X$ (= for every nbhd of $a$, the $\kappa$-sequence is eventually in that nbhd).  There is no assumption of continuity from $\kappa$ to $X$.

If $\kappa=\omega$, we get the usual notion of sequence and convergence of a sequence in $X$.  But viewed as an ordinal space, $\omega$ is discrete and every function from $\omega$ to $X$ is continuous.  So if we want to generalize to larger cardinals, we can consider transfinite sequences and their limits as in the paragraph above, or restrict attention to transfinite sequences that are continuous when $\kappa$ is given its usual ordinal topology.

That's basically the difference between UR and SUS: "unique radial limits" vs. "unique continuous radial limits".

(as an aside, if you want to put all this in pi-base, there are a lot of acronyms around, maybe already too many to add even more of them not sanctioned by the literature (yet).  Should UR be URL?  (remember Joel David Hamkins's comment in https://mathoverflow.net/questions/452903).  Should "SUS" be called "strongly US" in words?

Somehow I still feel uneasy about the SUS terminology.  The more natural generalization of "unique sequential limits" seems to be "unique radial limits" (UR or URL).  And "strongly US" does not tell me anything about being "unique continuous radial limits".  It actually sounds like it would be a variant of US itself (i.e. dealing with sequences instead of transfinite sequences).  Maybe UCR, or UCRL, or "continuously UR" or "continuously URL"?

**Reply by StevenClontz** [reply]
RE: put in pi-Base. Actually, I'm not certain that, right now, the UR/SUS concepts are notable enough to include in pi-Base. At least, not until they are published somewhere. That will also address the naming question: whatever ends up in the literature.

That said, if you're interested in collaborating on a paper on the topic, spinning out the discussions here, I'm up for that. I also invited M W to join this conversation if they are interested, particularly since they answered the big question. Since they are anonymous on Math.SE, there's nothing more to be done unless they join this conversation or email me.

That said, the choice of SUS is of course just a meme. I'm open to thinking more carefully about what should be used in a paper, although I'm fairly adamant about what I want the title to be: "Topological spaces among US" (with the joke not extending into the paper itself).

**Reply by StevenClontz** [reply]
I think I'm leaning *UCR* ("Unique Continuously Radial limits"), with the terminology of "transfinite sequences" vs. "continuously transfinite sequences". Here's the current map.

```
    /-----------> UR ------------\
  /                               v
T2 -> k1H -> ... -> k2H -> UK -> UCR -> US -> T1
```

I think we're missing examples for UR-not-UK (we have UR-not-k1H), and UK-not-k2H. Or, of course, theorems showing we cannot.

**Reply by prabau** [reply]
Agree with you that the concepts don't seem notable enough to get into pi-base at this point.  But maybe later.  As far as writing up a paper on the topic, M W should really be a coauthor if they are interested.  Not sure if I can contribute much at this point, but I can start thinking about it and if I can get something meaningful to add, I could be added as a collaborator.  And if nothing much gets added from my part, I don't need to be.

Great title "Topological spaces among US" by the way!  I also like UCR (Unique Continuously Radial limits).

**Reply by StevenClontz** [reply]
https://topology.pi-base.org/spaces/S000031/ is countable and US, so UR, but is not KC. It is wH though.

**Reply by StevenClontz** [reply]
~~Take the Tychonoff plank and double $(\omega_1,\omega)$. The only transfinite sequences converging to either copy of $(\omega_1,\omega)$ are eventually constant, and the deleted Tychonoff plank is radial and $T_2$, and it follows that we've consructed a UR space. But the space is not $UK$ as the deleted Tychonoff plank can be compactified by either copy of $(\omega_1,\omega)$.~~

This isn't right - we could take $\omega_1\times\omega$ with a doubled copy of $(\omega_1,\omega)$ and indeed have a UR space. But then $(\omega_1,\omega)$ no longer compactifies the space.

**Reply by marswill** [reply]
So I don't have an example for either UR-not-UK or UK-not-k2H yet. However, we can already do a little better than UR-not-KC, since the [one point compactification of the Arens-Fort space](https://topology.pi-base.org/spaces/S000165/properties) is countable and US, hence UR, and is not weak-Hausdorff.

I suspect UR-not-UK shouldn't be too hard.  The point-doubling argument will work if we can produce a compact Hausdorff space with a single radially inaccessible (non isolated) point.  That feels like something that should be easy enough to cook up, unless somehow it's impossible, though I haven't thought of the example yet - in pi-base the only compact hausdorff space that isn't radial is the Tychonoff plank, and that example doesn't quite work, since the one bad point is still radially accessible [edit: sorry, hadn't refreshed, I see Steven already covered that.]

I thought for awhile maybe $\beta\omega$ might be radially discrete but it seems like that's probably not true, as you could probably, given any nonprinciple ultrafilter $F$, take a maximal chain of filters ending in $F$, and produce a nontrivial transfinite sequence converging to $F$ from that.  Didn't work out the details but that seems likely to be dead end.

UK-not-K2H seems like a thornier question.  I actually wanted to find such an example in the stack exchange answer but couldn't come up with one, and will have to think more on how to approach that question.  Certainly if UK is equivalent to k2H that would be a nice surprise.

**Reply by StevenClontz** [reply]
I'm beginning to think UK might be to one-point compactifications what k2H is to Stone-Cech compactifications. Or put another way, UK focuses on a single compactifying point, but k2H handles all points. (EDIT: I'm less convinced of this today.)

**Reply by StevenClontz** [reply]
Prior art suggests the use of "strong" rather than "continuous" for radial/pseduoradial proprerties: https://arxiv.org/abs/1904.04416

**Reply by prabau** [reply]
Good find about "strong pseudoradial".  The 2015 article by Brazas & Fabel: https://arxiv.org/pdf/1301.4624.pdf

**Reply by StevenClontz** [reply]
The authors there use **unique strong pseudoradial convergence**. But is there a reason to use "pseudo"? I.e. what would **unique strong radial convergence* be?

**Reply by prabau** [reply]
I agree there is no reason to use "pseudo" in their terminology.  Presumably they were only interested in pseudoradial and not radial spaces in that paper, so they just carried over the "pseudo" prefix without thinking it through.

**Reply by StevenClontz** [reply]
I've updated most posts here to use **USR** for "Unique Strongly Radial limits" accordingly.

**Comment by StevenClontz** [comment]
Cleaning up definition of UK...

### Definition

A space is $X$ is **UK** if for every locally compact, non-compact Hausdorff space $L$ with continuous $f:L\to X$, there is at most one point $x\in X$ such that $f$ extended to $L$'s one-point compactification $K=L\cup\{\infty\}$ by $f(\infty)=x$ is continuous.

### Proposition

$k_2H \Longrightarrow UK \Longrightarrow USR$

**Proof:** For the first, take $L$, $f:L\to X$ continuous, and points $x_0,x_1$ such that extending $f$ to $K=L\cup\{\infty\}$ by either $f(\infty)=x_i$ is continuous. Define $g:K\times\{0,1\}\to X$ by $g(l,i)=f(l)$ and $g(\infty,i)=x_i$. Then $g$ is continuous, and there are no neighborhoods of $(\infty,0),(\infty,1)$ whose images are disjoint, so $g(\infty,0)=g(\infty,1)$, that is, $x_0=x_1$.

The second follows as every infinite cardinal is a locally compact, non-compact Hausdorff space.

**Comment by marswill** [comment]
We actually do indeed have $UR\implies UK$.  

To see this, we first note that every non-isolated point in a compact Hausdorff space is radially accessible (i.e., it is the limit of some transfinite sequence that is not eventually constant).  To prove this, let $X$ be compact Hausdorff, let $x\in X$ be non-isolated, and let $\mathcal C$ be a maximal chain of compact subsets of $X$ that contain $x$ as a non-isolated point, ordered by reverse inclusion.  Then $\bigcap \mathcal C$ is compact, and must contain $x$ as an isolated point (otherwise $\bigcap \mathcal C$ would be a maximal member of $\mathcal C$, but then we could intersect with a closed neighborhood of $x$ that omits some other member of $\bigcap \mathcal C$ to obtain a strictly smaller compact set still containing $x$ as a limit point.) 

But now, we can assume with no loss of generality that in fact $\bigcap \mathcal C=\\{x\\}$, since otherwise we could take a closed neighborhood $D$ of $x$ with $D\cap \bigcap \mathcal C=\\{x\\}$, and replace $\mathcal C$ with the family $\\{C\cap D\mid C\in\mathcal C\\}$.

Therefore, we may choose a well-ordered co-final chain $\\{C_\alpha\\}\subseteq \mathcal C$, and for each $\alpha$ choose $x_\alpha\in C_\alpha\backslash \\{x\\}$.  We know every open neighborhood $U$ of $x$ must contain some $C_\alpha$, since otherwise the family $\\{C_\alpha\backslash U\\}$ is a family of closed sets with the finite intersection property but empty intersection, contradicting compactness.  Thus $U$ contains $x_\beta$ for each $\beta>\alpha$, since $C_\beta\subset C_\alpha$.  As this holds for all neighborhoods, the sequence converges to $x$.

Finally, since every non-isolated point in a compact Hausdorff space is radially accessible, if $X$ is $UR$ and $f\colon Z\to X$ is a map from a locally compact noncompact Hausdorff space $Z$ with compactifying limits $x_1,x_2\in X$, then let $z_\alpha\in Z$ be a transfinite sequence converging to $\infty$ in $Z^+$.  Then $f(z_\alpha)$ converges to $x_1$ and $x_2$, so by $UR$, $x_1=x_2$.

**Reply by marswill** [reply]
>~Hmm, haven't been able to do serious research on this yet but I gather just from googling and inferring from how things are phrased that it is at least consistent that there are compact Hausdorff spaces that fail to be pseudoradial.  See e.g. this link http://at.yorku.ca/p/a/a/g/11.htm.  I'm sure with a bit more digging we can figure out whether this is always true in $ZFC$, or whether additional axioms are necessary.~

[Edit: Never mind the striken comment above, I forgot about $\beta\omega$ not being pseudoradial from [this question.](https://math.stackexchange.com/a/4834571/1210477), so the proof below (taking $Z=\beta\omega$ and $A=\omega$) actually shows that from $\beta\omega$ one can construct a $UR$ -not- $k_2H$ space.]

----------

In any case, we have $UR\implies k_2H$ if and only if every compact Hausdorff space is pseudoradial.

**Proof**

If every compact Hausdorff space is pseudoradial [edit this is never true in ZFC, but I'll leave the argument for posterity in case the logic is useful elsewhere], then suppose $X$ is $UR$, $Z$ is compact Hausdorff, and $f_1,f_2\colon Z\to X$ are continuous, and let $A\subseteq Z$ be the set on which $f_1=f_2$. Then for any transfinite sequence $a_\alpha\in A$ converging to $z\in Z$, $f_1(z)=\lim f_1(a_\alpha)=\lim f_2(a_\alpha)=f_2(z)$, so $z\in A$.  Thus $A$ is radially closed, so by pseudoradiallity of $Z$, $A$ is closed.  Since this holds for arbitrary compact Hausdorff $Z$, we see that $X$ is $k_2H$.

In the other direction, if $Z$ is compact Hausdorff and not pseudoradial, let $A\subset Z$ be radially closed but not closed.  Let $B=Z\backslash A$, and let $X=A\cup B_1\cup B_2$ where $B_1$ and $B_2$ are disjoint homeomorphic copies of $B$, via homeomorphisms $\pi_i\colon B_i\to B$.  Give $X$ the topology where $U\subseteq X$ is open if and only if $(U\cap A)\cup \pi_i(U\cap B_i)$ is open in $Z$, and let $\pi\colon X\to Z$ be such that $\pi$ is the identity on $A$ and agrees with $\pi_i$ on $B_i$.  Note that $\pi$ is continuous.

We also have continuous maps $f_i\colon Z\to X$, where $f_i$ restricts to the identity on $A$, and to $\pi_i^{-1}$ on $B$.

Then $X$ is $UR$.  To see this, observe that if $x_1\neq x_2\in X$, then $x_1$ and $x_2$ can always be separated by disjoint open neighborhoods, except in the case  $x_i=\pi_i(b)$ for some $b\in B$.  In this latter case, suppose $x_\alpha$ is a transfinite sequence converging to $x_1$.   Let 
$$S_\alpha:=\overline{\bigcup_{\beta>\alpha}\{\pi(x_\beta)\}}.$$
Since $A$ is radially closed in $Z$, we eventually have $S_\alpha\cap A=\emptyset$, since $S_\alpha$ eventually lies in every closed neighborhood of $b$, so if the intersection were always nonempty we could extract a radially convergent sequence of points in $A$ converging to $b\notin A$.   

Therefore we may choose some $\alpha$ with $S_\alpha\cap A=\emptyset$.  Since $X\backslash f_2(S_\alpha)$ is by definition a neighborhood of $x_1$, we eventually have $x_\beta\notin f_2(S_\alpha)$, so eventually $x_\beta\in B_1$, and thus $x_\beta\in f_1(S_\alpha)$.  But then $x_\beta\notin X\backslash f_1(S_\alpha)$, which is a neighborhood of $x_2$, so the sequence cannot converge to $x_2$.  We conclude that $X$ is $UR$.

On the other hand, the maps $f_i$ are continuous maps from the compact Hausdorff $Z$ into $X$, which agree precisely on the non-closed set $A$, and therefore witness that $X$ is not $k_2H$.

**Reply by marswill** [reply]
Also, since $UR\implies UK$ is true and  $UR\implies k_2H$ is false, that also gives us that $UK\implies k_2H$ is false, so that completely determines all implications between the properties, i.e., we have implications 

$$T_2\implies UR\implies UK,$$

$$T_2\implies k_1H\implies... k_2H\implies UK,$$

and

$$UK\implies USR \implies US\implies T_1,$$

all strict, and no other implications.

**Reply by prabau** [reply]
@marswill fyi, for braces "{" in math you need to precede them with double backslashes here.

**Comment by StevenClontz** [comment]
@marswill I'm going to DM you on the code4math Zulip to coordinate a time to hop on Zoom and move this forward.

**Comment by marswill** [comment]
We can generalize some of this discussion.  I'll outline the general concepts here, with the rigorous proofs TBA in the comments when I get a chance.

------------

 Let $\mathcal C$ be a class of topological spaces.  For a topological space $X$:

 - the **$\mathcal C$-ification $\mathcal CX$** is the set $X$, together with the final topology from the family of all continuous maps $f\colon Z\to X$ with $Z\in \mathcal C$.  
 - A subset of $X$ is **$\mathcal C$-closed** if it is closed in $\mathcal C X$.
 - $X$ is **$\mathcal C$-generated** ($\mathcal C$-Gen)if $\mathcal C X=X$ (equivalently, the closed subsets and $\mathcal C$-closed sets coincide).
 -  $X$ is **$\mathcal C$-Hausdorff** if the diagonal is $\mathcal C$-closed (equivalently, the diagonal in $\mathcal C(X\times X)$ is closed).



(@prabau had [a math se post](https://math.stackexchange.com/a/4761129/1210477) more or less outlining this approach for $k_n$-topologies, $n=1,2,3$, though only $n=1,2$ conform to what we're doing here).


------------------

Then consider the following classes:

 - $\mathcal{SIERP}$ consists solely of the Sierpinski space.
 - $\mathcal{SEQ}:=\\{\omega+1\\}$.
 - $\mathcal{ORD}$ is the class of ordinals with their order topology.
 - $\mathcal{K_H}$ is the class of compact Hausdorff spaces.
 - $\mathcal{K}$ is the class of all compact spaces.
 - $\mathcal{RAD}$ is the class of successors to limit ordinals, with discrete topology except at their maximum point, which has the same neighborhoods as the order topology. (We could in fact limit it to successors to regular cardinals and nothing changes).
 - $\mathcal{HAUS}$ is the class of all Hausdorff spaces.

Then a space $X$ is 
- $\text{Alexandrov} \iff \mathcal{SIERP}\text{-Gen}$. *
- $\text{Sequential}\iff  \mathcal{SEQ}\text{-Gen}$.
- $CG_2\iff  \mathcal{K_H}\text{-Gen}$.
- $CG_1\iff \mathcal{K}\text{-Gen}$.
- $\text{Pseudoradial}\iff \mathcal{RAD}\text{-Gen}$. 
- Every topological space is  $\mathcal{HAUS}\text{-Gen}$. *

Moreover, $X$ is
- $T_1\iff \mathcal{SIERP}\text{-Hausdorff}$. *
- $US\iff \mathcal{SEQ}\text{-Hausdorff}$.
- $USR\iff \mathcal{ORD}\text{-Hausdorff}$. *
- $k_2H\iff \mathcal{K_H}\text{-Hausdorff}$.
- $k_1H\iff \mathcal{K}\text{-Hausdorff}$.
- $UR\iff \mathcal{RAD}\text{-Hausdorff}$.
- $T_2\iff \mathcal {HAUS}\text{-Hausdorff}$. *

[*These will be elaborated on in comments.]



Note that $UK$ doesn't seem to fit into this picture, though there is a modifications to $UK$ that might be of interest that I'll try to spell out a little later.  Bottom line, though, while I don't have an absolute proof one way or another as to whether $UK$ is $\mathcal C$-Hausdorff for some class $\mathcal C$, I'm leaning against believing there is such a class.

----------------------------

We can generalize the proof from a previous comment, which said that $UR\implies k_2H$ if and only if every compact Hausdorff space is Pseudoradial (of course both statements are in fact false), as follows:

>  **Lemma.**  Let $\mathcal C$ and $\mathcal D$ be classes of topological spaces.  If every member of $\mathcal C$ is $\mathcal D$-generated, then every $\mathcal D$-Hausdorff space is $\mathcal C$-Hausdorff.  

[proof TBA in comments]



~Under a Hausdorff assumption we have a converse:~

>~**Proposition.** Let $\mathcal C$ and $\mathcal D$ be classes of topological spaces, so that every member of $\mathcal C$ is Hausdorff.  If there is some member of $\mathcal C$ that is not $\mathcal D$-generated, then there is a $\mathcal D$-Hausdorff space that fails to be $\mathcal C$-Hausdorff.~

~[Proof TBA in comments]~

~The previous result relating $UR$ and $k_2H$ is a special case of the preceding results, with $\mathcal C=\mathcal{K_H}$ and $\mathcal D=\mathcal{RAD}$.~

----------------------------------

**Update**

I should have known this was too good to be true.  After trying to write it up the proof of the proposition carefully, it became clear that in my proof above, I was relying much more that I realized on important properties of $\mathcal{RAD}$ that do not generalize.

I suspect we do have a much less ambitious version of the stricken proposition, though having learned my lesson, I will call this a conjecture until after I've carefully written the proof:

> **Conjecture.** Let $\mathcal C$ be a class of topological spaces.  If there is some $T_3$ member of $\mathcal C$ that is not $\mathcal{RAD}$-generated(i.e. Pseudoradial), then there is a $\mathcal{RAD}$-Hausdorff (i.e., $UR$) space that fails to be $\mathcal C$-Hausdorff.

I do feel it's pretty likely my argument does work for the aforementioned "conjecture," and will try to write it up soon. I also suspect it might generalize past transfinite sequences  to Fort spaces modeled on directed sets in a certain sense.

**Reply by StevenClontz** [reply]
I like this approach a lot.

**Reply by marswill** [reply]
First, let's tackle the starred statements.

>  $\text{Alexandrov} \iff \mathcal{SIERP}\text{-Gen}$. *

Let $S=\\{0,1\\}$ be the Sierpinski space with the topology where $1\in \overline{\\{0\\}}$.

$X$ is Alexandrov if and only if closed sets are precisely those for which 
$$x\in A\implies \overline{\\{x\\}}\subseteq A.\tag{1}$$ 
(I could dig up a reference but I think this is standard and is an exercise regardless).

(1) is equivalent to the statement that $A$ is closed if and only if for all $x,y\in X$, with $y\in\overline{\\{x\\}}$, we have $x\in A\implies y\in A$.  But continuous maps $f\colon S\to X$ are precisely those maps for which $f(1)\in \overline{\\{f(0)\\}}$, so every pair $x,y$ with  $y\in\overline{\\{x\\}}$ corresponds to a continuous map $f(0)=x$, $f(1)=y$, and vice versa, The statement $x\in A\implies y\in A$ is then equivalent to the statement that $0\in f^{-1}(A)\implies 1\in f^{-1}(A)$, which in turn is precisely equivalent to the claim that $f^{-1}(A)$ is closed. 


> - Every topological space is  $\mathcal{HAUS}\text{-Gen}$. *

If $A\subseteq X$ is not closed, then let $p\in\overline{A}\backslash A$.  There are two possibilities:

If $p\in \overline{\\{a\\}}$ for some $a\in A$, then define $f\colon [0,1]\to X$ by 
 
$$f(t)=
\begin{cases}
a & t<1\\
p & t=1.
\end{cases}
$$

Then $f$ is continuous and $f^{-1}(A)=[0,1)$ is not closed. 

If, on the other hand, we have that $p\notin \overline{\\{a\\}}$ for each $a\in A$, then let $Z=A\cup \\{p\\}$ have the topology where the neighborhoods of $p$ are precisely those neighborhoods from the subspace topology in $X$, and every other singleton is open.  Then since $p\notin \overline{\\{a\\}}$ for each $a\in A$, $Z$ is $T_1$ with all but one point isolated, hence $Z$ is $T_2$, and the inclusion map from $Z$ into $X$ is continuous, and is our desired map.

>  $T_1\iff \mathcal{SIERP}\text{-Hausdorff}$. *

If $X$ is $T_1$, then any continuous map $f\colon S\to X$ must satisfy $f(1)\in \overline{\\{f(0)\\}}=\\{f(0)\\}$, hence are constant.  Thus any two such maps agree precisely on either all of $S$ or $\emptyset$, so the diagonal is $\mathcal{SIERP}$-closed.  Conversely, if $X$ fails to be $T_1$, then let $y\neq x$, $y\in \overline{\\{x\\}}$.  Then $f(0)=g(0)=x$, $f(1)=x$, $g(1)=y$ defines two continuous functions which agree on a non-closed set, so the diagonal is not  $\mathcal{SIERP}$-closed. 

> $USR\iff \mathcal{ORD}\text{-Hausdorff}$. *

If $X$ fails to be $USR$, then let $f,g\colon \alpha+1\to X$ be two continuous maps that agree for all $\beta<\alpha$, and disagree on $\alpha$.  Then they agree precisely on a non-closed subset, so the diagonal is not $\mathcal{ORD}$-closed.  
Conversely, suppose $f,g\colon \alpha+1\to X$ are two maps that agree precisely on a non-closed subset  $A\subset\alpha+1$.  Then let $\beta$ be the first element of $\overline{A}\backslash A$.  

Then $\beta$ is certainly a limit ordinal, and $\overline{A}\cap (\beta+1)$ is a closed subset of $\alpha+1$ with a maximal element (namely $\beta$), hence homeomorphic to the successor of a limit ordinal.  Since the restrictions of $f$ and $g$ agree on every element of $\overline{A}\cap (\beta+1)$ except $\beta$, they witness a contradiction to $USR$.

> $T_2\iff \mathcal {HAUS}\text{-Hausdorff}$. *

Since $\mathcal{HAUS}\text{-Gen}$ consists of all topological spaces, the $\mathcal{HAUS}$-ification of a space is the space itself, so $\mathcal{HAUS}$-Hausdorff spaces are simply those in which the diagonal is closed.

**Reply by marswill** [reply]
**Proof of Lemma.**

Suppose every member of $\mathcal C$ is $\mathcal D$-generated, and let $X$ be $\mathcal D$-Hausdorff.  Let $f,g\colon Z\to X$ be continuous, with $Z\in \mathcal C$, and let $A=\\{z\in Z\mid f(x)=g(x)\\}$.  Then for every continuous $h\colon Y\to Z$, with $Y\in \mathcal D$, we have $h^{-1}(A)=\\{y\in Y\mid f\circ h(y)=g\circ h(y)\\}$.  Since $Y\in \mathcal D$ and $X$ is $\mathcal D$-Hausdorff, it follows that $h^{-1}(A)$, as the set of points on which $f\circ h$ and $g\circ h$ agree, is closed.  

Since this holds for all such maps $h$, and $Z$ as a member of $\mathcal C$ is $\mathcal D$-generated, we see that $A$ is closed.  As this applies to arbitrary pairs of maps $f,g\colon Z\to X$, with $Z\in \mathcal C$, we conclude the diagonal of $X\times X$ is $\mathcal  C$-closed.

**Comment by prabau** [comment]
I have not read [this](https://www.ams.org/journals/tran/1969-146-00/S0002-9947-1969-0252473-2), but you may find it possibly somewhat related?
